doctrine/doctrine-migrations-bundle/1.2/etc/packages/migrations.yaml

```diff
@@ -0,0 +1,2 @@
+doctrine_migrations:
+    dir_name: '%kernel.project_dir%/DoctrineMigrations'
```

Comment:
@fabpot do you think migration classes should go in `src` or `etc` ? Your previous comment suggests `etc`, but migrations are code.